app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -25,7 +25,7 @@ class ProjectBoard extends React.Component {
           ${I18n.t("projects.show.in_progress")}`}>
           <Sprints
             stories={this.props.columns.backlog.stories}
```

Comment:
Do we still need the stories in this component now?

Reply:
We need the stories to verify if there is no stories and `return null`, at `Sprints` component: 
```
const Sprints = ({ stories, sprints }) => {
  if (!stories.length) return null;

  return <div className="Sprints">{renderSprints(sprints)}</div>;
};
```
https://github.com/Codeminer42/cm42-central/pull/384/files#diff-5d5553a06db9db79b688faad27adc356R34

Reply:
@julliasaad @talyssonoc in this case, can't we just check if sprints is empty instead?

Reply:
At first, sprints will come **empty** and then will be filled, so I cannot just verify if sprints are empty.

Reply:
My mistake, I added `sprints: []` at initialState in Sprints component and now we can use sprints to check ;)